compositor_render/src/state/render_loop.rs

```diff
@@ -106,7 +109,10 @@ pub(super) fn read_outputs(
                 }
                 OutputTexture::Rgba8UnormWgpuTexture { resolution } => {
                     let wgpu_texture = match ctx.wgpu_ctx.mode {
-                        RenderingMode::GpuOptimized | RenderingMode::WebGl => {
+                        RenderingMode::GpuOptimized => {
+                            RgbaMultiViewTexture::new(ctx.wgpu_ctx, *resolution).texture_owned()
+                        }
+                        RenderingMode::WebGl => {
                             RgbaSrgbTexture::new(ctx.wgpu_ctx, *resolution).texture_owned()
```

Comment:
We need textures that have `unorm` view format. `WebGl` does not matter here because vulkan won't work here either way